src/main/java/com/google/songgame/servlets/RoomServlet.java

```diff
@@ -60,11 +60,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   public void doPut(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Map<String, String> roomProperties = readJSONFromRequest(request);
 
-    // Room query that looks for correct room.
-    Query roomQuery =
-        new Query("Room").addFilter("roomId", FilterOperator.EQUAL, roomProperties.get("roomId"));
-    PreparedQuery result = datastore.prepare(roomQuery);
-    Entity currentRoom = result.asSingleEntity();
+    // Get correct room.
```

Comment:
I think comments like this one and the same one in your doGet method aren't necessary now that we have the loadRoom() method, since it's apparent what the code is doing based on the method name, and at that point the comment is superfluous. Even worse, in the future someone may update the code but forget to update the comment. This is kind of a special case of the DRY principle, in which you don't want your comments to simply repeat what is obvious from the code.

To quote an internal best practices guide about comments:

"What comments are for
Use a comment when it is infeasible to make your code self-explanatory. Here's a great quote from Robert C. Martin's Clean Code:

The proper use of comments is to compensate for our failure to express ourself in code. Note that I used the word failure. I meant it. Comments are always failures. We must have them because we cannot always figure out how to express ourselves without them, but their use is not a cause for celebration.... Every time you express yourself in code, you should pat yourself on the back. Every time you write a comment, you should grimace and feel the failure of your ability of expression.

Self-explanatory code beats liberally-commented code (which, of course, beats code that can't be understood). If you need a comment to explain what a block of code does, try to introduce an explaining variable or to extract a method. If you've already done that but you still need a comment, try to rename it until it answers all your concerns. Sometimes, just choosing better identifier names can be enough; sometimes it takes more work. It's almost always worth it."

I think it's worth going through your changes in this PR and removing comments when what they describe is apparent from the code they are attached to.

Reply:
@pruess-google cool I removed those comments, and another that was pretty self explanatory. Thanks!